lib/blocks/latest-posts.php

```diff
@@ -43,8 +48,10 @@ function gutenberg_render_block_core_latest_posts( $attributes ) {
 		$posts_content .= "<li><a href='{$post_permalink}'>{$post_title}</a></li>\n";
 	}
 
+	$class = 'blocks-latest-posts ' . esc_attr( 'align' . $align );
```

Comment:
We should follow the same convention as the frontend blocks to generate the classname. This should be `wp-block-latest-posts`